src/kirby/kirby.common.ts

```diff
@@ -50,8 +54,15 @@ export const declarations = [
   AvatarComponent,
   GroupByPipe,
   SpinnerComponent,
+  ModalComponent,
```

Comment:
Not needed (you don't use it declarative, you use it as an entry component)
```suggestion
```

Reply:
this does not compile unfortunately